main/src/hadd-argparse.py

```diff
@@ -34,6 +34,16 @@ def get_argparse():
         "compression setting after fk (for example 206 when using -fk206)"), action = 'store_true')
     parser.add_argument("-ff", help="The compression level use is the one specified in the first input", action = 'store_true')
     parser.add_argument("-k", help="Skip corrupt or non-existent files, do not exit", action = 'store_true')
+    parser.add_argument("-L", help=textwrap.fill(
+       "Read the list of objects from FILE and either only merge or skip those objects depending on "
```

Comment:
This is a powerful new features but does not match the title of the PR.

The title talks about filtering which files are being merged together while the feature documented here and implemented below is about filtering which objects and `TDirectories` within the files are actually merged (but does not affect the list of files opened).

Reply:
See `tutorials/io/mergeSelective.C`

Reply:
@pcanal the PR title was indeed wrong; I changed "files" to "objects".